tests/functional/l/logging/logging_fstring_interpolation_py37.py

```diff
@@ -3,3 +3,9 @@
 
 VAR = "string"
 logging.error(f"{VAR}")  # [logging-fstring-interpolation]
+
+WORLD = "world"
+logging.error(f'Hello {WORLD}')  # [logging-fstring-interpolation]
```

Comment:
For consistency we could use the `"` instead of `'` like `VAR` above. Perhaps we should run black on the test files 🤔

Reply:
pylint need to be able to handle simple quote as well as double quote so the tests needs to stay diverse. If everything is perfectly blacked then we don't know if we handle non blacked code correctly.

Reply:
yeah most of the functional test modules are not black-ened

Reply:
👍